Specs/Scene/OctahedralProjectedCubeMapSpec.js

```diff
@@ -14,7 +14,7 @@ describe(
     var octahedralMap;
```

Comment:
Miscellaneous comments:

* Do a global search for `ktx`, `crn`, and `crunch` (with `Match Whole Word`). Remove all occurrences of `crn`/`crunch` (except in `CHANGES.md`), and see if `ktx` needs to be renamed to `ktx2` anywhere.
* Update `CHANGES.md` with a breaking changes section for 1.82 for `loadCRN` and `loadKTX` being removed
* I wonder if there's any way to build `Source/ThirdParty/basis_transcoder.wasm` so that it's smaller...

Reply:
@ebogo1 bump to finish doing the first bullet here

Reply:
I found some leftover occurrences: https://github.com/CesiumGS/cesium/pull/9513/commits/804f1ebdb695f62d46f3e8da1c74695dc1fb74e3